src/fparser/tests/test_parser.py

```diff
@@ -71,9 +71,18 @@
 """
 
 import pytest
-from fparser.block_statements import *
-from fparser.readfortran import Line, FortranStringReader
-from nose.tools import assert_equal
+from fparser.block_statements import Allocatable, Allocate, ArithmeticIf, \
+     AssignedGoto, Assign, Assignment, Asynchronous, Backspace, Bind, Call, \
+     CallProtoArgument, CallStatement, Character, Check, Close, Common, \
+     ComputedGoto, Contains, Continue, Cycle, Data, Deallocate, Depend, \
+     Dimension, Else, ElseIf, ElseWhere, Endfile, Entry, Enumerator, \
+     Equivalence, Exit, External, FinalBinding, Flush, Format, ForallStmt, \
+     FortranName, GenericBinding, Goto, Implicit, Import, Inquire, Integer, \
+     Intent, Intrinsic, ModuleProcedure, Nullify, Open, Optional, Parameter, \
+     Pointer, PointerAssignment, Pause, Print, Private, Protected, Public, \
+     Read, Return, Rewind, Save, Sequence, SpecificBinding, Stop, Target, \
+     Threadsafe, Use, Value, Volatile, Wait, WhereStmt, Write
```

Comment:
Does it make sense to replace `from something import *` with `from something import yadda, blah, and_whatever` if `yadda`, `blah`, and `and_whatever` is equal to `*`? In this case I think deactivating the pylint message with a reasonable comment would be fine, too.

Reply:
I got recently bitten by a kind of import * (Fortran, ... but still) , so I decided to go this way - even though believe me I found it highly annoying ;) For me the big advantage is that you can search for any symbol and see where it comes from. Alternative we could use
```
import fparser.block_statements
```
et. All in all, I admit I don't care, if the maintainers prefer import * that's fine with me, too.

Reply:
That's why we are doing `use xxx, only: yyy` everywhere :) I was just wondering whether there are other reasons besides that.

Reply:
*grin* except that using ```only``` confused PSyclone (ticket to be opened there as soon as I have a test case), so I had to remove the ```only```. And that module defined 'min' as a variable - believe me, I thought I was going crazy, esp  ;)
No, no other reason - except that I don't now exactly what the maintainers prefer, so I played it safe ;)

Reply:
I prefer explicit!